rest/src/main/java/pro/taskana/rest/RestApplication.java

```diff
@@ -40,23 +48,40 @@ public static void main(String[] args) {
     }
 
     @Bean
-    public ClassificationService getClassificationService() throws Exception {
-        return getTaskanaEngine().getClassificationService();
+    @Primary
+    @ConfigurationProperties(prefix = "datasource")
+    public DataSourceProperties dataSourceProperties() {
+        return new DataSourceProperties();
     }
 
     @Bean
-    public TaskService getTaskService() throws Exception {
-        return getTaskanaEngine().getTaskService();
+    @Primary
+    public DataSource dataSource(DataSourceProperties properties) {
+        DataSource dataSource = properties.initializeDataSourceBuilder().build();
+        // if TaskanaEngineImpl runs with SpringManagedTransactionFactory, then
```

Comment:
you can remove this comment or make it more clear to use TransactionAwareDataSourceProxy if using native jdbc api..